verticapy/ml_pipeline/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
```

Comment:
Is it really ml_pipeline? I think those pipeline can be used for different type of project. Let's create a "pipeline" folder.